File: LeetCode-Problems/Easy/Search-Insert-Position.py
# Python
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        for x in range(0, len(nums)):
            if (nums[x] == target):
                return x
        y = 0
        while (y < len(nums)):
            if (nums[y] < target):
                y = y + 1
            else:
                return y
        return y

File: LeetCode-Problems/Easy/Search-Insert-Position.java
//Java
class Solution {
    public int searchInsert(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++){
            if (target == nums[i]){
                return i;
            }
        }
        int x = 0;
        while (x < nums.length){
            if (nums[x] < target){
                x++;
            }
            else{
                return x;
            }
        }
        return x;
    }
}
